packages/protect/src/ffi/operations/encrypt.ts

```diff
@@ -67,7 +67,9 @@ export class EncryptOperation
         }
 
         if (this.plaintext === null) {
-          return null
+          return {
+            data: null,
+          }
```

Comment:
How should nulls handled with the composite type @tobyhede? Should the whole value be null, or should the inner `data` field be `null`?

Reply:
Either are valid, but I think the whole value should be null. 
A `NULL` data field will reduce down to `ROW(NULL)`.